src/System Application/App/File System/app.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id":  "c9c54414-80c3-4cc9-98c6-589158882774",
+    "name":  "File System",
```

Comment:
I'm still not sure happy about the module name. It's not a file system. It's a way to use different cloud file service providers through a unified API. Maybe we should ask Copilot for fitting name ;-)

Reply:
What do you think of File Service or File Storage Service?

Reply:
I do like the ring of File Storage Service, but in reality it's not a service. It's a framework to leverage services. The module itself doesn't connect to any service itself. It helps you organize your file services. It's too early to be creative, but I'll get back to you later with a suggested name or two 😊

But maybe you're on to something. Also considering how you right now would call the module:

```
FileSystem.Initialize(Enum::"File Scenario"::"Order Confirmation");
FilePath := FileSystem.CombinePath(FSSetup."Folder to Store", StrSubstNo(PDFNameLbl, SalesHeader."No."));       
if not FileSystem.FileExists(FilePath) then
    FileSystem.CreateFile(FilePath, Stream);
```

It kind of makes sense to call it FileStorageService! Maybe I'll come around 😊

Reply:
I discussed the name with @darjoo, who suggested to maybe just call the module File Storage. To me that would make sense, as this isn't a file system nor service, but it's an API to access storage systems. The code would then look like this:

```
FileStorage.Initialize(Enum::"File Scenario"::"Order Confirmation");
if not FileStorage.FileExists(FilePath) then
    FileStorage.CreateFile(FilePath, Stream);
```

I like that! What do you think, @IceOnly?

Reply:
I would prefer FileStorage. If there is a native BC solution such as One Drive or something similar in the future, it can also provide an interface.

But in the end you are the ones responsible for the product. Just give me a module name and a namespace and I'll rename everything.

Reply:
When we add a native storage service, it will go through the platform in one way or another. I'd like to be explicit here, that this module is for external providers. Let's go with "External File Storage".

Reply:
In some cases, 30 characters were simply not enough.
I tried Ext. as preifx if > 30 characters. But even that wasn't always possible, I hope it fits thanks to the namespace.

Reply:
We could maybe have settled for Ext. Storage", but I think we'll be alright with the solution you picked!